pipelines/environments/mldesigner-2.0-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
+
+ENV AZUREML_CONDA_ENVIRONMENT_PATH /azureml-envs/mldesigner
+
+# Create conda environment
+RUN conda create -p $AZUREML_CONDA_ENVIRONMENT_PATH \
+    python=3.8.13 pip=21.3.1 -c anaconda -c conda-forge
+
+# Prepend path to AzureML conda environment
+ENV PATH $AZUREML_CONDA_ENVIRONMENT_PATH/bin:$PATH
+
+# Install pip dependencies
+# update mldesigner to macro after its pre-release complete, currently manually update it
+RUN pip install mldesigner \
```

Comment:
Please pin the `mldesigner` version by using `=={{latest-pypi-version}}` or hard-coding it here. This will help reproducibility.

Reply:
Hard-code this for now